src/app/service/constants/constants.ts

```diff
@@ -8,6 +8,7 @@ export default Object.freeze({
 	DB_COLLECTION_TIMECARDS: 'timecards',
 	DB_COLLECTION_GUEST_USERS: 'guestUsers',
 	DB_COLLECTION_BOUNTIES: 'bounties',
+	DB_DISCORD_USER_ACCOUNTS: 'discordUserAccounts',
```

Comment:
what is the best collection for us to store users' wallet addresses?

Reply:
Let's keep that data with DB_COLLECTION_DISCORD_USERS, in this collection we keep track of all discord users and we can expand to keep track of future settings and options

Reply:
updated ✅